routes/project/user.rb

```diff
@@ -176,15 +181,67 @@ class Clover
       r.post "policy/managed" do
         authorize("Project:user", @project.id)
 
-        invitation_policies = r.params["invitation_policies"] || {}
-        allowed_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:add").select_hash(:name, Sequel.as(true, :v))
-        invitation_policies.each do |email, policy|
-          if allowed_tags[policy]
-            @project.invitations.find { _1.email == email }&.update(policy: policy)
+        DB.transaction do
+          allowed_add_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:add").all.to_h { [_1.name, _1] }
+          allowed_remove_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:remove").all.to_h { [_1.name, _1] }
+          removals = Hash.new(0)
+          additions = Hash.new(0)
+          changes = []
+          issues = []
+          user_policies = r.params["user_policies"] || {}
+          user_policies.each do |ubid, policy|
+            policy = nil if policy == ""
+            user = Account.from_ubid(ubid)
+            existing_tags = user.subject_tags_dataset.where(project_id: @project.id).select_map(:name)
+            next unless existing_tags.length < 2
```

Comment:
This case should add to `issues`, with a message like: "Cannot change policy for user, as they are in multiple subject tags"